sdmetrics/reports/single_table/_properties/column_shapes.py

```diff
@@ -48,14 +46,14 @@ def _generate_details(self, real_data, synthetic_data, metadata, progress_bar=No
                     column_score = metric.compute(
                         real_data[column_name], synthetic_data[column_name]
                     )
+                    error_message = None
                 else:
                     continue
 
             except Exception as e:
                 column_score = np.nan
-                warning_messages.append(
-                        f"Unable to compute Column Shape for column '{column_name}'. "
-                        f'Encountered Error: {type(e).__name__} {e}'
+                error_message = (
```

Comment:
One thing that could be potentially nice for us is to log the stacktrace if possible. Since we are just blindly catching exceptions, it is sometimes hard to debug